filters/include/pcl/filters/impl/box_clipper3D.hpp

```diff
@@ -108,21 +108,23 @@ pcl::BoxClipper3D<PointT>::transformPoint (const PointT& pointIn, PointT& pointO
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 // ToDo use product on point.getVector3fMap () and transformatio_.col (i) to use the SSE advantages of eigen
+// http://www.opengl-tutorial.org/beginners-tutorials/tutorial-3-matrices/#an-introduction-to-matrices
```

Comment:
Please remove this comment, I don't feel it's necessary here. Why do you felt it was particularly important to include it?